base/applications/cmdutils/more/more.c

```diff
@@ -59,6 +198,12 @@ PagePrompt(PCON_PAGER Pager, DWORD Done, DWORD Total)
     HANDLE hInput = ConStreamGetOSHandle(StdIn);
     DWORD dwMode;
     KEY_EVENT_RECORD KeyEvent;
+    BOOL fCtrl;
+    DWORD nLines;
+    WCHAR chSubCommand = 0;
```

Comment:
The chSubCommand can now be moved within the function that uses it.

Reply:
OK.